docs/Images.md

```diff
@@ -34,15 +34,15 @@ And `button.js` code contains
 <Image source={require('./img/check.png')} />
 ```
 
-Packager will bundle and serve the image corresponding to device's screen density, e.g. on iPhone 5s `check@2x.png` will be used, on Nexus 5 – `check@3x.png`. If there is no image matching the screen density, the closest best option will be selected.
+Packager will bundle and serve the image corresponding to the device's screen density, e.g. on iPhone 5s `check@2x.png` will be used, on Nexus 5 – `check@3x.png`. If there is no image matching the screen density, the closest best option will be selected.
```

Comment:
I’m not a native speaker, but wouldn’t “to the screen density of the device” be even better.